docs/content-services/components/tree.component.md

```diff
@@ -0,0 +1,110 @@
+---
+Title: Tree component
+Added: v6.0.0-A.1
+Status: Active
+Last reviewed: 2023-01-25
+---
+
+# [Tree component](../../../lib/content-services/src/lib/tree/components/tree.component.ts "Defined in tree.component.ts")
+
+Shows the nodes in tree structure, each node containing children is collapsible/expandable. Can be integrated with any datasource extending [Tree service](../../../lib/content-services//src/lib/tree/services/tree.service.ts). 
+
+![Tree component screenshot](../../docassets/images/tree.png)
+
+## Basic Usage
+
+```html
+<adf-tree
+    [displayName]="'Tree display name'"
+    [loadMoreSuffix]="'subnodes'"
+    [emptyContentTemplate]="emptyContentTemplate"
+    [nodeActionsMenuTemplate]="nodeActionsMenuTemplate"
+    (paginationChanged)="onPaginationChanged($event)">
+</adf-tree>
+```
+
+## Class members
+
+### Properties
+
+| Name | Type | Default value | Description |
+| ---- | ---- | ------------- | ----------- |
+| emptyContentTemplate | `TemplateRef` |  | Template that will  be rendered when no nodes are loaded. |
+| nodeActionsMenuTemplate | `TemplateRef` |  | Template that will  be rendered when context menu for given node is opened. |
+| stickyHeader | `boolean` | false | If set to true header will be sticky. |
+| selectableNodes | `boolean` | false | If set to true nodes will be selectable. |
+| displayName | `string` | | Display name for tree title. |
+| loadMoreSuffix | `string` | | Suffix added to `Load more` string inside load more node. |
+| expandIcon | `string` | `chevron_right` | Icon shown when node is collapsed. |
+| collapseIcon | `string` | `expand_more` | Icon showed when ndoe is expanded. |
+| treeNodesSelection | `SelectionModel<T>` | | Selection model with selected tree nodes. |
```

Comment:
That field has not Input decorator so probably should not be listed here?

Reply:
Right, removed